README.md

```diff
@@ -256,12 +255,17 @@ Python 3.x, nginx, uwsgi and sqlite and should be sufficient for a few users
     See the [uWSGI documentation](http://uwsgi-docs.readthedocs.io/en/latest/)
     for more advanced configuration details.
 
-    **Note: Find the location of the pipenv virtual environment for the 
+    **Note: Find the location of the pipenv virtual environment for the
     `virtualenv` parameter with the command `pipenv --venv`.**
 
+    Symlink it and restart:
```

Comment:
I like this idea! It does make the flow of the steps much cleaner. But, please change this to treat the "Symlink it and restart" commands as a new step. I think that will make it even cleaner.